test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -240,17 +240,17 @@ public void PackCommand_NewSolution_ContinuousOutputInDefaultPaths()
                     ProjectFileUtils.WriteXmlToFile(xml, stream);
                 }
 
-                _dotnetFixture.RestoreSolutionExpectSuccess(testDirectory, solutionName, testOutputHelper: _testOutputHelper);
-                _dotnetFixture.BuildSolutionExpectSuccess(testDirectory, solutionName, testOutputHelper: _testOutputHelper);
+                _dotnetFixture.RunDotnetExpectSuccess(testDirectory, $"restore {solutionName}.slnx", testOutputHelper: _testOutputHelper);
+                _dotnetFixture.RunDotnetExpectSuccess(testDirectory, $"build {solutionName}.slnx -c Release", testOutputHelper: _testOutputHelper);
```

Comment:
Could you help me understand why this is changing to Release configuration from Debug?

Reply:
Here is the [doc for this behavior change](https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/dotnet-pack-config). I am surprised that we have not hit this because the docs say that this change was made in .NET 8 SDK.